### DISCUSSION TITLE
How to change the hot key to custom.(not just the m1-4 but all f1-12)

### DISCUSSION THREAD

**Discussion by pauljojy** [seerge/g-helper#3269] [Q&A]
is it possible to change all the f1-12 keys to custom hotkeys. like i would like too change screenshot one to play/pause in g14 (2024).

**Comment by seerge** [comment]
@pauljojy hello, if you want just to rebind one standard windows key to another you can use [Sharpkeys](https://github.com/randyrants/sharpkeys) :)

**Reply by pauljojy** [reply]
this works for me. but if u could add the feature of customizing them in ghelper it would reduce the background activities. Since ghelper already does support customization for some hotkeys
